### PR TITLE
[TACHYON-769] remove sticky-bit-setting on files

### PR DESCRIPTION
**Pull request by chen-tian**
https://tachyon.atlassian.net/browse/TACHYON-769

Also rename setLocalFileStickyBit to setLocalDirStickyBit

### CODE REVIEW

File: common/src/main/java/tachyon/util/io/FileUtils.java [hunk 1]
```diff
@@ -102,20 +102,20 @@ public static void changeLocalFileToFullPermission(String filePath) throws IOExc
...
     try {
       // sticky bit is not implemented in PosixFilePermission
-      if (file.startsWith(TachyonURI.SEPARATOR)) {
-        Runtime.getRuntime().exec("chmod o+t " + file);
+      if (dir.startsWith(TachyonURI.SEPARATOR)) {
+        Runtime.getRuntime().exec("chmod o+t " + dir);
       }
     } catch (IOException e) {
-      LOG.info("Can not set the sticky bit of the file : " + file);
+      LOG.info("Can not set the sticky bit to the direcotry : " + dir);
```
Comment (calvinjia) [inline common/src/main/java/tachyon/util/io/FileUtils.java]: Could you change `to` back to `of`?

**Comment by calvinjia** [comment]
@chen-tian Thanks for the patch, 1 minor comment, overall looks good.

**Comment by calvinjia** [comment]
+1 LGTM

**Comment by apc999** [comment]
+1 LGTM

**Comment by fanshiqing** [comment]
Actually, the comments on method `setLocalDirStickyBit(String)` in this PR is unclear and a bit misleading. I have improved it in PR:https://github.com/amplab/tachyon/pull/1270
Can one of the admins merge it? Many thanks!

**Pull request merged**